### DISCUSSION TITLE
德國 LRZ 將 CS-2 AI 系統引入「未來運算」計畫

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#664] [General]
https://technews.tw/2022/06/09/lrz-cs-2-ai/
LRZ（Leibniz-Rechenzentrum）是德國三大超級運算中心之一，其餘兩個是斯圖加特高效能運算中心（HLRS）、於利希超級運算中心（JSC）。運算中心是位於慕尼黑的超級計算中心，主要提供科研機構、學區研究、網路運營商與雲端服務供應商等運算軟體與套件工具、數據管理、伺服器虛擬託管等服務。

**Comment by treeann** [comment]
AI的用途真的太廣泛了
